aggressive-indent.el

```diff
@@ -487,7 +487,7 @@ If BODY finishes, `while-no-input' returns whatever value BODY produced."
 ;;; Minor modes
 ;;;###autoload
 (define-minor-mode aggressive-indent-mode
-  nil nil " =>"
+  :lighter " =>"
```

Comment:
This is incorrect. The docstring was missing here.

Reply:
Please see #157.